OneBlink.SDK/FormsClient.cs

```diff
@@ -339,6 +339,50 @@ public async Task<AttachmentData> CreateSubmissionAttachment(long formId, Stream
             return attachmentData;
         }
 
+        public async Task<EmailAttachmentData> UploadEmailAttachment(string filename, string contentType, Stream body)

```

Comment:
Docs missed for this https://github.com/oneblink/sdk-dotnet/blob/master/docs/forms-client.md